art/__main__.py

```diff
@@ -103,3 +108,7 @@ def select_test(test_name="TEST"):
             help_func()
     else:
         help_func()
```

Comment:
As code factor mentioned, this function is too long.

Reply:
I mentioned this issue in our discord server

Reply:
fixed in https://github.com/sepandhaghighi/art/pull/220/commits/dc8bf888060bfca9911c46ed6d691f4531477837